flink-clients/src/main/java/org/apache/flink/client/program/Client.java

```diff
@@ -68,7 +68,7 @@
 	
 	private final Optimizer compiler;		// the compiler to compile the jobs
 	
-	private boolean printStatusDuringExecution = false;
```

Comment:
Is this an intentional change? Doesn't it change the behavior of all submission clients based on this class?